Client/game_sa/CWeatherSA.cpp

```diff
@@ -228,3 +228,25 @@ bool CWeatherSA::ResetRainbow()
     MemCpy((LPVOID)(0x72BF59 + 2), &originalCodes, 3);
     return true;
 }
+
+bool CWeatherSA::SetTimerCycle(unsigned char value) noexcept
```

Comment:
`bool` + `noexcept`?

Reply:
what do you mean

Reply:
> what do you mean

```cpp
bool CWeatherSA::SetTimerCycle(bool value) noexcept
```

Reply:
I used unsigned char to make it compatible with SetWorldProperty which accepts float, to avoid refactoring

Reply:
It accepts float, but you're passing it to a byte (unsinged char)?

Reply:
I will  refactoring some of SetWorldProperty with std::variant to hold float and boolean which is better